fedimint-core/src/encoding/tbs.rs

```diff
@@ -97,6 +97,42 @@ impl Decodable for tbs::BlindingKey {
     }
 }
 
+impl Encodable for threshold_crypto::Ciphertext {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        self.to_bytes().consensus_encode(writer)
+    }
+}
+
+impl Decodable for threshold_crypto::Ciphertext {
```

Comment:
Just FYI, I needed to do something similar for `threshold_crypto::PublicKey` in https://github.com/fedimint/fedimint/pull/2279